modules/assim.sequential/R/Analysis_sda_multiSite.R

```diff
@@ -417,20 +439,23 @@ GEF.MultiSite<-function(setting, Forecast, Observed, H, extraArg,...){
     conf$addMonitors(c("X","Xall","q","Xs")) 
     samplerNumberOffset <<- length(conf$getSamplers())
     
-    for(i in 1:length(y.ind)) {
-      node <- paste0('y.censored[',i,']')
-      conf$addSampler(node, 'toggle', control=list(type='RW'))
-    }
-    
+      for(i in 1:length(y.ind)) {
```

Comment:
Why the extra indent?

Reply:
No reason, I spent probably 2 weeks just debugging this section of the function.